docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+      depends_on:
+      - db
+
+      db:
+      image: postgres:16.0-alpine3.17
+      restart: always
+      env_file:
+      - .env
+      ports:
+      - "5432:5432"
+      volumes:
+      - my_db:$PGDATA

```

Comment:
The volume `my_db:$PGDATA` uses an environment variable `$PGDATA`. Ensure that this variable is defined in your environment or `.env` file to avoid issues with volume mounting.